python/cpenv_ui/dialogs.py

```diff
@@ -77,9 +77,17 @@ def set_frame(self, text):
 
     def set_progress(self, chunk_size=None, max_size=None):
         if max_size:
+            if max_size != 1:
+                if len(str(max_size)) > 9:
+                    # Fix for OverflowError int larger than maximum 32-bit integer
+                    # not super elegant, but it works
+                    max_size = max_size / 1000
             self.progress.setRange(0, max_size)
```

Comment:
Looks like we doubled up on some lines of code here. If you patch this, happy to merge!